api/v1alpha1/multiclusterservice_types.go

```diff
@@ -186,8 +186,14 @@ type ServiceUpgradePaths struct {
 	Namespace string `json:"namespace"`
 	// Template is the name of the current service template.
 	Template string `json:"template"`
-	// UpgradePaths contains details for the state of service upgrade paths.
-	UpgradePaths [][]string `json:"upgradePaths,omitempty"`
+	// AvailableUpgrades contains details for the state of available upgrades.
+	AvailableUpgrades []UpgradePath `json:"availableUpgrades,omitempty"`
+}
+
+// UpgradePath contains details for the state of service upgrade paths.
+type UpgradePath struct {
+	// Versions contains the list of versions that service can be upgraded to.
+	Versions []string `json:"upgradePaths,omitempty"`
```

Comment:
Should this be `Name` or `TemplateName` instead because it is currently holding values such as `foo-0-2-0` etc, which are template names (not versions).